README.md

```diff
@@ -8,7 +8,7 @@
 
 #### Quicklinks
 
-[![Python 3.9](https://img.shields.io/pypi/pyversions/raiden.svg)](https://raiden-network.readthedocs.io/en/stable/)  [![Chat on Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/raiden-network/raiden?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+[![Python 3.9](https://img.shields.io/pypi/pyversions/raiden.svg)](https://raiden-network.readthedocs.io/en/stable/)  [![Chat on Discord](https://img.shields.io/discord/948623129796825109.svg)](https://discord.com/channels/948623129796825109/)
```

Comment:
Wouldn't it be better to have the invite as the link? I'm not quite sure about how the link to the server id will work here.